content/blog/2024/testing-fastapi.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Testing FastAPI Applications
+author: ed
+date: 2024-02-08
+tags:
+  - Article
+  - Python
+  - Testing
+  - FastAPI
+# image:
+#   src:
```

Comment:
I couldn't find an image on Unsplash that made sense for this article. Any suggestions?

Reply:
You could do something with "light fixture"-
https://unsplash.com/photos/white-and-black-light-fixture-_z0DiiaIhB4
https://unsplash.com/photos/white-light-bulb-turned-on-in-room-Ir3aiqw1QHM
https://unsplash.com/photos/lighted-bulbs-on-ceiling-TAUybipul4Q